addons/account/models/account_reconcile_model.py

```diff
@@ -231,134 +230,121 @@ def _get_taxes_move_lines_dict(self, tax, base_line_dict):
                 'account_id': tax_res['account_id'] or base_line_dict['account_id'],
                 'name': tax_res['name'],
                 'partner_id': base_line_dict.get('partner_id'),
-                'debit': tax_res['amount'] > 0 and tax_res['amount'] or 0,
-                'credit': tax_res['amount'] < 0 and -tax_res['amount'] or 0,
+                'balance': tax_res['amount'],
                 'analytic_account_id': tax.analytic and base_line_dict['analytic_account_id'],
                 'analytic_tag_ids': tax.analytic and base_line_dict['analytic_tag_ids'],
                 'tax_exigible': tax_res['tax_exigibility'],
                 'tax_repartition_line_id': tax_res['tax_repartition_line_id'],
                 'tax_ids': [(6, 0, tax_res['tax_ids'])],
                 'tax_tag_ids': [(6, 0, tax_res['tag_ids'])],
+                'tax_tag_ids': [(6, 0, tax_res['tag_ids'])],
+                'currency_id': False,
                 'reconcile_model_id': self.id,
             })
 
             # Handle price included taxes.
-            base_line_dict['debit'] = tax_res['base'] > 0 and tax_res['base'] or base_line_dict['debit']
-            base_line_dict['credit'] = tax_res['base'] < 0 and -tax_res['base'] or base_line_dict['credit']
+            base_line_dict['balance'] = tax_res['base']
```

Comment:
?

Reply:
Same here: this dictionary will be passed to the `reconcile` method in `account.bank.statement.line`.